packages/ra-core/src/controller/list/useListController.storeKey.stories.tsx

```diff
@@ -0,0 +1,129 @@
+        { id: 3, title: 'Post #3', votes: 30 },
+        { id: 4, title: 'Post #4', votes: 40 },
+        { id: 5, title: 'Post #5', votes: 50 },
+        { id: 6, title: 'Post #6', votes: 60 },
+        { id: 7, title: 'Post #7', votes: 70 },
+    ],
+});
+
+const listControllerComponent = (storeKey: string, sort?: SortPayload) => {
+    return (
+        <ListController
+            resource="posts"
+            debounce={200}
+            perPage={3}
+            sort={sort}
+            storeKey={storeKey}
+            children={(params: ListControllerResult) => (
+                <div>
+                    <span aria-label="storeKey" data-value={storeKey}>
+                        storeKey: {storeKey}
+                    </span>
+                    <br />
+                    <span aria-label="perPage" data-value={params.perPage}>
+                        perPage: {params.perPage}
+                    </span>
+                    <br />
+                    <br />
+                    <button
+                        aria-label="incrementPerPage"
+                        disabled={params.perPage > params.data?.length ?? 0}
+                        onClick={() => params.setPerPage(++params.perPage)}
+                    >
+                        Increment perPage
+                    </button>{' '}
+                    <button
+                        aria-label="decrementPerPage"
+                        disabled={params.perPage <= 0}
+                        onClick={() => params.setPerPage(--params.perPage)}
+                    >
+                        Decrement perPage
+                    </button>
+                    <ul style={styles.ul}>
+                        {!params.isLoading &&
+                            params.data.map(post => (
+                                <li key={`post_${post.id}`}>
+                                    {post.title} - {post.votes} votes
+                                </li>
+                            ))}
+                    </ul>
+                </div>
+            )}
+        />
+    );
+};
+
+const MinimalLayout = (props: CoreLayoutProps) => {
+    return (
+        <div style={styles.mainContainer}>
+            <Link aria-label="top" to={`/top`}>
+                Go to Top
+            </Link>{' '}
+            <Link aria-label="flop" to={`/flop`}>
+                Go to Flop
+            </Link>
+            <br />
+            <br />
+            {props.children}
+        </div>
+    );
+};
+const TopList = () =>
+    listControllerComponent('top', { field: 'votes', order: 'DESC' });
+const FlopList = () =>
+    listControllerComponent('flop', { field: 'votes', order: 'ASC' });
+
+export const ListsUsingSameResource = (argsOrProps, context) => {
+    const history = context?.history || argsOrProps.history;
+    return (
+        <CoreAdmin
+            history={history}
+            layout={MinimalLayout}
+            store={localStorageStore()}
```

Comment:
Actually, even though `CoreAdmin` accepts a `store` prop, it doesn't use it internally (this should be fixed in TS, but it's another topic).
So currently your story is actually storing the list params _in memory_ and not in local storage. :grimacing: 

I believe it would be better to fully demonstrate this feature if you could add another story where we actually use the local storage. Maybe all you need is to replace `CoreAdmin` by `AdminContext`, but I'm not 100% sure...

Reply:
Or your could maybe simply wrap it in a `StoreContextProvider`